holisticai/bias/plots/_bias_report_plots.py

```diff
@@ -0,0 +1,543 @@
+# base imports
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+# utils
+from holisticai.utils import get_colors
+
+
+def bias_report_regression(
+    metrics: pd.DataFrame, metrics_mitigated: pd.DataFrame = None
+):
+    """
+    Plot bias report for regression models.
+
+    Parameters
+    ----------
+    data : pandas.DataFrame
+        Dataframe containing bias metrics.
+    mitigated : bool, optional
+        Whether the bias metrics are for mitigated model or not, by default False
+    """
+    metric_names = list(metrics.index)
+
+    cols = 4
+    rows = len(metric_names) // cols
+    if len(metric_names) % cols != 0:
+        rows += 1
+
+    fig, axes = plt.subplots(ncols=cols, nrows=rows, figsize=(15, 7))
+
+    if metrics_mitigated is None:
+        for i, name in enumerate(metric_names):
+            metrics_biased = metrics.copy()
+            metrics_biased.columns = ["Baseline", "Reference"]
+            metric_data = metrics_biased[metrics_biased.index == name]
+            row, col = divmod(i, cols)
+            axes[row, col].bar(
+                metric_data.index,
+                metric_data["Baseline"],
+                label="Baseline",
+                color=get_colors(1)[0],
+            )
+            axes[row, col].set_title(name)
+            axes[row, col].axhline(
+                y=metric_data["Reference"].values[0], color="black", linestyle="--"
+            )
+            axes[row, col].set_ylabel("Score")
+            axes[row, col].set_xticklabels([])
+
+            if i == len(metric_names) - 1 and i % cols != cols - 1:
+                for j in range(i % cols + 1, cols):
+                    axes[row, j].axis("off")
+            elif i == len(metric_names) - 1 and i % cols == cols - 1:
+                for j in range(cols):
+                    axes[row + 1, j].axis("off")
+
+            if name == "RMSE Ratio":

```

Comment:
store the range values in a dictionary (reduce this code)

Reply:
Done.